src/test/java/io/stargate/sgv2/jsonapi/service/operation/collections/InsertCollectionOperationTest.java

```diff
@@ -83,15 +83,17 @@ public List<CollectionInsertAttempt> createInsertAttempt(
 
   static final String INSERT_CQL =
       "INSERT INTO \"%s\".\"%s\""
-          + " (key, tx_id, doc_json, exist_keys, array_size, array_contains, query_bool_values, query_dbl_values , query_text_values, query_null_values, query_timestamp_values)"
+          + " (key, tx_id, doc_json, exist_keys, array_size, array_contains, query_bool_values,"
+          + " query_dbl_values, query_text_values, query_null_values, query_timestamp_values)"
           + " VALUES"
-          + " (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)  IF NOT EXISTS";
```

Comment:
Same here, remove extra space to unify CQL statements (for now, MUST match char-by-char for tests)